### PR TITLE
feat: added dlss-support command

### PR DESCRIPTION
**Pull request by nakajimayoshi**
feat: added dlss-support command

## Description
Adds a command to explain the new DLSS feature added to msfs. 

## Test Results

image
## Discord Username
HowNowBrownCrow#2591

### CODE REVIEW

**Comment by tracernz** [comment]
It's particularly noticable with display quality. Everything else looks pretty good.

**Comment by oim1** [comment]
make sure you update the command docs file as well

**Comment by NathanInnes** [comment]
What do you mean by "mileage may vary"

**Comment by nakajimayoshi** [comment]
> What do you mean by "mileage may vary"

Sounds too colloquial in my opinion so I'll change it. RTX cards have differing amounts of tensor cores which are used to perform the upscaling. Someone with a 2060 isn't going to see the same performance results as someone with a 3090ti.

**Comment by NathanInnes** [comment]
> > What do you mean by "mileage may vary"
> 
> 
> 
> Sounds too colloquial in my opinion so I'll change it. RTX cards have differing amounts of tensor cores which are used to perform the upscaling. Someone with a 2060 isn't going to see the same performance results as someone with a 3090ti. 

RIP. I got a 2060 but yeah maybe look into wording it differently if you wish to keep that in

**Comment by benw202** [comment]
Is this still in the works @nakajimayoshi?

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Update <small>_ July 2022</small>
 - refactor: birthday command (11/07/2022)
 - feat: default ATC command (10/07/2022)
 - fix: remove flypados thread from .exp (07/07/2022)
+- refactor: weather.ts embed title and copy (07/12/2022)
```
Comment (benw202) [inline .github/CHANGELOG.md:49]:
```suggestion
- refactor: weather.ts embed title and copy (07/12/2022)
```

I am not sure why this has ended up here?

**Comment by benw202** [comment]
Image has been approved - will hold merge for your decision on if to include :)

**Comment by benw202** [comment]
Staging has been merged in to make use of new check.

I have bumped the date forwards as the changelog entry for Vals PR is the 10th as he is ahead and this will be merged 10th or later haha!

**Comment by benw202** [comment]
Please add this to `/.github)/command-docs.md` as it is a new command

**Comment by nakajimayoshi** [comment]
> Please add this to `/.github)/command-docs.md` as it is a new command

@benw202 Done ✅

File: src/commands/support/dlss.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export const dlss: CommandDefinition = {
+    name: 'dlss',
+    description: 'Explanation of the DLSS option in the MSFS graphics settings.',
+    category: CommandCategory.SUPPORT,
+    executor: async (msg) => {
+        const dlssEmbed = makeEmbed({
+            title: 'MSFS | NVIDIA DLSS',
+            description: makeLines([
+                'Deep Learning Super Sampling (DLSS) is a graphics technology that renders content at a lower resolution, then upscales it to improve FPS. ',
```
Comment (NathanInnes) [inline src/commands/support/dlss.ts:13]: Extra space character here and also line 15

File: src/commands/support/dlss.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                '',
+                'Note that this will affect display quality. ',
+                '',
+                'You need to have an Nvidia GeForce RTX card to use the setting, and your results may vary with your hardware setup when using the feature.',
+                '',
+                'See [Nvidia\'s page on DLSS](https://www.nvidia.com/en-us/geforce/technologies/dlss/) for more information.',
+            ]),
+        });
+
+        await msg.channel.send({ embeds: [dlssEmbed] });
```
Comment (NathanInnes) [inline src/commands/support/dlss.ts:23]: No need to async await, return will do :)

**Review by NUTTTTTTTT — APPROVED** [review]

**Comment by benw202** [comment]
I totally forgot and didn't say in time before this was merged. Can the image be added to the assets folder please

**Pull request merged**